frontend/src/features/settings/types/SpecialAuthorization.ts

```diff
@@ -1,25 +1,27 @@
-import { Nullable } from "../../../common/types/common";
+import { Nullable, RequiredOrNull } from "../../../common/types/common";
 import { PermitType } from "../../permits/types/PermitType";
 
 export const NO_FEE_PERMIT_TYPES = {
-  GOV_PROV_CAN: 1,
-  MUNICIPAL_CAN: 2,
-  DISTRICT_OUT_BC: 3,
-  GOV_USA: 4,
-  GOV_PROV_USA: 5,
+  CA_GOVT: "CA_GOVT",
+  MUNICPALITY: "MUNICPALITY",
+  SCHOOL: "SCHOOL",
+  USA_GOVT: "USA_GOVT",
+  ANY_USA_GOVT: "ANY_USA_GOVT",
```

Comment:
What's the difference between `USA_GOVT` and `ANY_USA_GOVT`?

Reply:
They actually mean different things (different no-fee permit types). USA_GOVT refers to "the USA government", and ANY_USA_GOVT refers to "any state or county in the USA".

Reply:
I think
- `USA_GOVT` or `USA_FEDERAL_GOVT` 
- `OTHER_USA_GOVT` 
would be a better names here.